From f51eb5cc7c8ac5b23dec1348a7c4cf3f6b6b24eb Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Mon, 13 Jul 2015 20:36:59 +0800
Subject: [PATCH] MLK-11241: vadc: Add s_mbus_fmt interface

mx6s capture driver will check the return value
of s_mbus_fmt, captuer driver will failed if subdev
driver no s_mbus_fmt function.
so add s_mbus_fmt function to vadc subdev driver.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 drivers/media/platform/mxc/subdev/mxc_vadc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/mxc/subdev/mxc_vadc.c b/drivers/media/platform/mxc/subdev/mxc_vadc.c
index d3bd549..1c0fbe1 100644
--- a/drivers/media/platform/mxc/subdev/mxc_vadc.c
+++ b/drivers/media/platform/mxc/subdev/mxc_vadc.c
@@ -566,6 +566,7 @@ static const struct v4l2_subdev_video_ops vadc_video_ops = {
 	.enum_mbus_fmt         = vadc_enum_mbus_fmt,
 	.try_mbus_fmt          = vadc_mbus_fmt,
 	.g_mbus_fmt            = vadc_mbus_fmt,
+	.s_mbus_fmt            = vadc_mbus_fmt,
 	.enum_framesizes       = vadc_enum_framesizes,
 	.enum_frameintervals   = vadc_enum_frameintervals,
 	.s_parm                = vadc_s_parm,
-- 
1.8.0

